spring-data-cassandra/src/main/resources/org/springframework/data/cassandra/config/spring-cassandra-1.0.xsd

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
There's also `spring-cql-1.0.xsd` in `spring-cql` which declares nearly the same elements. We should change the other one as well until we get to the point to keep only one of the two.